### PR TITLE
Text style tweaks for iuse.cpp.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
Some grammar and phrasing tweaks to improve understandability, spacing
fixes, capitalization standardization for mycus speech*, capitalization
for the M in Molotov cocktail.

*All but one of the (repeating) mycus lines was properly capitalized, so now
all of the mycus lines are capitalized.

### CODE REVIEW

**Comment by DavidKeaton** [comment]
Question: Molotov Cocktail, no? The entirety of both words is the proper noun, isn't it?

**Comment by Rivet-the-Zombie** [comment]
Nope, just the first part - *Molotovin koktaili* being the name given to them originally by the folks who invented them - those awesome European rednecks, the Finns - during the Winter War. The name references Soviet foreign minister Vyacheslav Molotov. During the war, they also euphemistically called cluster bombs 'Molotov bread baskets' (because apparently everyone in Finland has a really grim sense of humor) and a 'Molotov cocktail' is a nice 'beverage' to go with the Soviet-supplied 'food'. Molotov's name became attached to these things due to his involvment in the Molotov–Ribbentrop Pact (peace agreement between the CCCP and the NSDAP) which included secret provisions that divided up a bunch of other people's land (Romania, Poland, Lithuania, Latvia, Estonia, and Finland) between Россия and Deutschland, setting the stage for the Winter War, the September Campaign, Operation Tannenberg, the Katyn massacre, and many other horrible events which led to WWII.

**Comment by DavidKeaton** [comment]
Damn.

**Comment by KA101** [comment]
>Rivet infodumps

I'd heard it as Molotov claiming the bombs were really food drops for the poor starving Finns, but yeah, that's basically the size of it.

File: src/iuse.cpp [hunk 1]
```diff
@@ -1609,7 +1609,7 @@ int iuse::mutagen(player *p, item *it, bool, point)
 {
     if (p->has_trait("MUTAGEN_AVOID")) {
          //~"Uh-uh" is a sound used for "nope", "no", etc.
-        p->add_msg_if_player(m_warning, _("After what happened that last time? uh-uh.  You're not drinking that chemical stuff."));
+        p->add_msg_if_player(m_warning, _("After what happened that last time?  Uh-uh.  You're not drinking that chemical stuff."));
```
Comment (KA101) [inline src/iuse.cpp]: The non-caps were intentional in these lines: the uh-uh follows immediately after the question, so fast that it's not a new sentence.

File: src/iuse.cpp [hunk 2]
```diff
@@ -2089,7 +2089,7 @@ int iuse::marloss(player *p, item *it, bool t, point pos)
                         pgettext("memorial_female", "Opened the Marloss Gateway."));
         p->add_msg_if_player(m_good, _("You wake up in a marloss bush.  Almost *cradled* in it, actually, as though it grew there for you."));
         //~ Beginning to hear the Mycus while conscious: that's it speaking
-        p->add_msg_if_player(m_good, _("unity. together we have reached the door. we provide the final key. now to pass through..."));
+        p->add_msg_if_player(m_good, _("Unity.  Together we have reached the door.  We provide the final key.  Now to pass through..."));
```
Comment (KA101) [inline src/iuse.cpp]: Much like the slimespawns and the shared-brain communication, the Mycus is a small voice here.  Once you go full Mycus, sure, it's capitalized.

Comment (Rivet-the-Zombie) [inline src/iuse.cpp]: Right on. I'll revert these lines and add a note to reflect that.

**Comment by DavidKeaton** [comment]
I accept this as a fair and square "school was in session foo"

**Comment by Rivet-the-Zombie** [comment]
Meh. I'm a history nerd.

**Comment by DavidKeaton** [comment]
Either way, thumbs up!

**Pull request merged**